extra/release_tool.py

```diff
@@ -112,8 +117,18 @@ def set_custom_component_maps(self, maps):
     @staticmethod
     def _initialize_component_maps():
         if Component.COMPONENT_MAPS is None:
-            with open(os.path.join(integration_dir(), "component-maps.yml")) as fd:
-                Component.COMPONENT_MAPS = yaml.safe_load(fd)
+            if Component._integration_version:
+                component_maps = execute_git(
+                    None,
+                    integration_dir(),
+                    ["show", f"{Component._integration_version}:component-maps.yml"],
+                    capture=True,
+                    capture_stderr=True,
```

Comment:
Btw, it occurred to me that this should probably be false, so that if you mistype the version you can actually see it, instead of getting a very cryptic parsing error.